recipe/meta.yaml

```diff
@@ -26,6 +27,9 @@ requirements:
     - numpy ==1.23.5
     - pandas ==1.5.3  # 2.0.0 requires a newer version of xarray
     - dask =={{ dask_version }}
+{% if python_version in ["3.8", "3.9"] -%}
+    - dask-snowflake
+{%- endif %}
```

Comment:
Initially I didn't have this clause here, and CI broke trying to install requirements:

https://github.com/coiled/benchmarks/actions/runs/4875870950/jobs/8698634340

I'm pretty sure it's because `snowflake-connector-python` pins `numpy` to `<1.25.0`:

https://github.com/snowflakedb/snowflake-connector-python/blob/97c25993eb3fb43e0df634fd41f98f9e9ebf20bd/setup.cfg#L85

Lower version of `numpy` can't run on `python>3.9`.

CI still breaks though, there must be something else I'm missing.

Reply:
Definitely looking forward to https://github.com/coiled/benchmarks/pull/793!

Reply:
There is something weird going on though, the job is pulling numpy 1.19, which indeed isn't provided for python 3.10, but snowflake-connector-python allows 1.24 for example, which is the newest released version of numpy, something else seems to restrict numpy